app/src/main/java/net/gsantner/markor/frontend/NewFileDialog.java

```diff
@@ -231,6 +233,24 @@ private AlertDialog.Builder makeDialog(final File basedir, final boolean allowCr
             return TextViewUtils.interpolateSnippet(format, title, "").trim();
         };
 
+        final @ColorInt int color = titleEdit.getCurrentTextColor();
```

Comment:
Just a little thing which makes the text colour red if the currently entered file name already exists